.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = tab
+indent_size = 4
```

Comment:
Is `indent_size = 4` optional? As the `indent_style` is set to tab there is no need to forcefully overwrite the user's setting for that. 🙂